### PR TITLE
Vote-693: Move hard coded string into json file

### PR DESCRIPTION
**Pull request by clmedders**
Purpose: This Pr will move all hard coded strings into a separate json file to be maintained 
Ticket: [Vote-693](https://cm-jira.usa.gov/browse/VOTE-693)
Testing: Work your way though the app and ensure there is no regression... also spot check files to see if any were left out. 

NOTE: title attributes would not allow for this method.. the DOM would render the `sting.text` and not the text that it needed

### CODE REVIEW

File: src/components/FormSections/PersonalInfo.jsx [hunk 1]
```diff
@@ -189,7 +190,7 @@ function PersonalInfo(props){
                         >
                             <div data-testid="formGroup" className="usa-form-group usa-form-group--month">
                                 <label data-testid="label" className="usa-label" htmlFor="date_of_birth_month">
-                                    Month
+                                <span dangerouslySetInnerHTML= {{__html: string.month}}/>
```
Comment (rayestrada) [inline src/components/FormSections/PersonalInfo.jsx:193]: we do not need the dangerouslysetinnerhtml
set to `{string.month}`

File: src/components/FormSections/PersonalInfo.jsx [hunk 2]
```diff
@@ -213,7 +214,7 @@ function PersonalInfo(props){
                             </div>
                             <div data-testid="formGroup" className="usa-form-group usa-form-group--day">
                                 <label data-testid="label" className="usa-label" htmlFor="date_of_birth_day">
-                                    Day
+                                <span dangerouslySetInnerHTML= {{__html: string.day}}/>
```
Comment (rayestrada) [inline src/components/FormSections/PersonalInfo.jsx:217]: we do not need the dangerouslysetinnerhtml
set to `{string.day}`

File: src/components/FormSections/PersonalInfo.jsx [hunk 3]
```diff
@@ -237,7 +238,7 @@ function PersonalInfo(props){
                             </div>
                             <div data-testid="formGroup" className="usa-form-group usa-form-group--year">
                                 <label data-testid="label" className="usa-label" htmlFor="date_of_birth_year">
-                                    Year
+                                <span dangerouslySetInnerHTML= {{__html: string.year}}/>
```
Comment (rayestrada) [inline src/components/FormSections/PersonalInfo.jsx:241]: we do not need the dangerouslysetinnerhtml
set to `{string.year}`

File: src/strings.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+
+  {
+    "notRequired": "Not required for your state",
+    "newWindow": "Open form in a new window",
+    "nameChange": "I have legally changed my name since I last registered in this state.",
+    "dob": "Date of Birth",
+    "month": "Month",
+    "day": "Day",
+    "year": "Year",
+    "stateName": "Go to the PDF form on %state_name%'s website",
```
Comment (rayestrada) [inline src/strings.json:10]: Update this placeholder to be `@state_name` to be consistent.

File: src/strings.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "checkReg": "Check your registration",
+    "learnMore": "Learn more about your voting options",
+    "backBtn": "Back to Vote.gov",
+    "stateOnlineName": "Go to %state_name%'s online form",
```
Comment (rayestrada) [inline src/strings.json:14]: Update this placeholder to be `@state_name` to be consistent.

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: src/strings.json (hunk 4 above)
Comment (clmedders) [reply src/strings.json:10]: updated to @state_name

File: src/strings.json (hunk 5 above)
Comment (clmedders) [reply src/strings.json:14]: updated to @state_name

File: src/components/FormSections/PersonalInfo.jsx (hunk 1 above)
Comment (clmedders) [reply src/components/FormSections/PersonalInfo.jsx:193]: removed the dangerouslysetinnerhtml and fixed format

File: src/components/FormSections/PersonalInfo.jsx (hunk 2 above)
Comment (clmedders) [reply src/components/FormSections/PersonalInfo.jsx:217]: removed the dangerouslysetinnerhtml and fixed format

File: src/App.jsx [hunk 6]
```diff
@@ -8,6 +8,7 @@ import {fetchData, sanitizeDOM} from './components/HelperFunctions/JsonHelper.js
 import { HelmetProvider } from "react-helmet-async";
 import {getFieldValue} from "./components/HelperFunctions/fieldParser";
 import {GridContainer} from "@trussworks/react-uswds";
+import * as string from './strings.json';
```
[the remark this answers is not in the document]
Comment (clmedders) [reply src/App.jsx]: Update file and path for future work

File: src/strings.json (hunk 4 above)
Comment (clmedders) [reply src/strings.json]: updated

File: src/strings.json (hunk 5 above)
Comment (clmedders) [reply src/strings.json]: updated

**Review by rayestrada — APPROVED** [review]

**Pull request merged**